### PR TITLE
Add number of NIO loops created by Vert.x in the documentation.

### PR DESCRIPTION
**Pull request by poiuytrez**
I was very weird to noticed in production to have a lot of eventloop threads. I did not noticed that in development because I have less cores. It should be written in documentation to avoid WTF moments :).

Signed-off-by: Guillaume Charhon <guillaume@databerries.com>

### CODE REVIEW

File: src/main/asciidoc/java/index.adoc [hunk 1]
```diff
@@ -362,9 +362,10 @@ with respect to concurrency, scaling and deployment.
 
 To use this model, you write your code as set of *verticles*.
 
-Verticles are chunks of code that get deployed and
-run by Vert.x. Verticles can be written in any of the languages that Vert.x supports and a single application
-can include verticles written in multiple languages.
+Verticles are chunks of code that get deployed and run by Vert.x. Vert.x will create N event loops threads
```
Comment (purplefox) [inline src/main/asciidoc/java/index.adoc]: I would be clearer to say "A Vert.x instance maintains N event loop threads (where N by default is core*2) by default"

File: src/main/asciidoc/java/index.adoc [hunk 2]
```diff
@@ -362,9 +362,10 @@ with respect to concurrency, scaling and deployment.
...
+(where N by default is core*2) by default. When you deploy a verticle,
+it will be deployed on one of theses event loops. Verticles can be written in any of the languages that Vert.x supports
```
Comment (purplefox) [inline src/main/asciidoc/java/index.adoc]: The verticle is not deployed _on_ an event loop. It is _associated_ to an event loop when the verticle is created.

Also, this is only true for standard (i.e.non worker) verticles.

**Comment by poiuytrez** [comment]
Thank you @purplefox for the comments. I have corrected the documentation.

**Comment by cescoffier** [comment]
@poiuytrez you are not editing the right file. This file is generated. Can you update the package-info.java file ?

**Comment by poiuytrez** [comment]
@cescoffier I have updated the package-info.java file.

**Pull request merged**